docs/getting-started/mixed/index.md

```diff
@@ -0,0 +1,232 @@
+          servicePort: 80
+```
+
+Run the following commands to deploy:
+
+* A Rollout with the Linkerd `linkerd.io/inject: enabled` annotation
+* Two Services (stable and canary)
+* An Ingress
+
+```shell
+kubectl apply -f https://raw.githubusercontent.com/argoproj/argo-rollouts/master/docs/getting-started/mixed/rollout.yaml
+kubectl apply -f https://raw.githubusercontent.com/argoproj/argo-rollouts/master/docs/getting-started/mixed/services.yaml
+kubectl apply -f https://raw.githubusercontent.com/argoproj/argo-rollouts/master/docs/getting-started/mixed/ingress.yaml
+```
+
+After applying the manifests you should see the following rollout, services, and ingress resources
+in the cluster:
+
+```shell
+$ kubectl get ro
+NAME            DESIRED   CURRENT   UP-TO-DATE   AVAILABLE
+rollouts-demo   1         2         1            2
+
+$ kubectl get svc
+NAME                   TYPE        CLUSTER-IP       EXTERNAL-IP   PORT(S)   AGE
+rollouts-demo-canary   ClusterIP   10.111.69.188    <none>        80/TCP    23m
+rollouts-demo-stable   ClusterIP   10.109.175.248   <none>        80/TCP    23m
+
+$ kubectl get ing
+NAME                   CLASS    HOSTS                 ADDRESS        PORTS   AGE
+rollouts-demo-stable   <none>   rollouts-demo.local   192.168.64.2   80      23m
+```
+
+You should also see a TrafficSplit resource which is created automatically and owned by the rollout:
+
+```
+$ kubectl get trafficsplit
+NAME            SERVICE
+rollouts-demo   rollouts-demo-stable
+```
+
+When inspecting the generated TrafficSplit resource, the weights are automatically configured to
+send 100% traffic to the `rollouts-demo-stable` service, and 0% traffic to the `rollouts-demo-canary`.
+These values will be updated during an update.
+
+```yaml
+apiVersion: split.smi-spec.io/v1alpha1
+kind: TrafficSplit
+metadata:
+  name: rollouts-demo
+  namespace: default
+spec:
+  backends:
+  - service: rollouts-demo-canary
+    weight: "0"
+  - service: rollouts-demo-stable
+    weight: "100"
+  service: rollouts-demo-stable
+```
+
+You should also notice a second ingress created by the rollouts controller,
+`rollouts-demo-rollouts-demo-stable-canary`. This ingress is the "canary ingress", which is a
+clone of the user-managed Ingress referenced under `nginx.stableIngress`. It is used by nginx
+ingress controller to achieve canary traffic splitting. The name of the generated ingress is
+formulated using `<ROLLOUT-NAME>-<INGRESS-NAME>-canary`. More details on the second Ingress are
+discussed in the following section.
+
+## 2. Perform an update
+
+Now perform an update the rollout by changing the image, and wait for it to reached the paused state.
+
+```shell
+kubectl argo rollouts set image rollouts-demo rollouts-demo=argoproj/rollouts-demo:yellow
+kubectl argo rollouts get rollout rollouts-demo
+```
+
+![Rollout Paused](../nginx/paused-rollout-nginx.png)
+
+At this point, both the canary and stable version of the Rollout are running, with 5% of the
```

Comment:
> At this point, both the canary and stable

Can you please confirm is this sentence is correct? It seems to me that what happens is that at this point both smi and nginx are receiving 5% of traffic in their canary pods?

Reply:
noted.

Reply:
Interesting, I got it from `docs/getting-started/nginx/index.md` but your suggestion make sense.

Reply:
ammended.